src/Akka.HealthCheck/Configuration/akka.healthcheck.conf

```diff
@@ -3,6 +3,15 @@
 ############################################
 
 akka.healthcheck{
+
+	# Log the complete configuration at INFO level when the actor system is started.
+	# This is useful when you are uncertain of what configuration is used.
+	log-config-on-start = on
```

Comment:
Are these settings being parsed into the `AkkaHealthCheckSettings` class anywhere?